src/Stripe.net/Entities/Invoices/Invoice.cs

```diff
@@ -103,6 +103,44 @@ internal object InternalCustomer
         }
         #endregion
 
+        /// <summary>
+        /// The customer’s address. Until the invoice is finalized, this field will equal
+        /// <code>customer.address</code>. Once the invoice is finalized, this field will no longer
```

Comment:
```suggestion
        /// <c>customer.address</c>. Once the invoice is finalized, this field will no longer
```

`<code>` is for multiple lines of code, cf. https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/code-inline.